includes/abstracts/abstract.llms.post.model.php

```diff
@@ -1678,6 +1681,32 @@ public function toArray() {
 		return apply_filters( "llms_{$this->model_post_type}_to_array", $arr, $this );
 	}
 
+	/**
+	 * Enqueues provider scripts for the URL.
+	 *
+	 * @since [version]
+	 *
+	 * @param string $url URL to check.
+	 * @return null If no provider is found.
+	 */
+	public function get_provider_support( $url ) {
+
+		$host = wp_parse_url( $url, PHP_URL_HOST );
+
+		// VideoPress Provider.
+		if ( is_plugin_active( 'jetpack-videopress/jetpack-videopress.php' ) ) {
+			if ( strpos( $host, 'videopress.com' ) !== false || strpos( $host, 'video.wordpress.com' ) !== false ) {
+				wp_enqueue_script( 'videopress-token-bridge', plugins_url() . '/jetpack-videopress/jetpack_vendor/automattic/jetpack-videopress/src/../build/lib/token-bridge.js', array(), llms()->version, true );
```

Comment:
I'm not exactly sure why we need to specify the complete URL here, but it works!